### ISSUE TITLE
Use colors from Xresources / xrdb

### ISSUE THREAD

**Issue by blueyed** [kovidgoyal/kitty#647]
I think kitty should use the colors from the xrdb (Xresources) by default (if available), which would allow to use already configured themes without changing the config, or setting them via base16-shell or similar.

I am using base16-shell to setup terminal colors (and also update the global xrdb), but this is not taken into account for `edit_config_file` for example, since it execs kitty without the outer shell (basically `kitty kitty +runpy 'os.execvp("vim", ["vim"])'`, right?).

**Comment by xdrie** [comment]
for anyone who ends up here wanting a quick fix: [here](https://gist.github.com/xdrie/aabe3b7507683001baf7762df005890b) is such a script.

**Comment by Ahwxorg** [comment]
@redthing1, do you know of a way to do this in bash? i dont particularly like adding yet another script to my ~/.local/bin

**Comment by Ahwxorg** [comment]
also, just a heads up: the link links to your old github username.

**Comment by redthing1** [comment]
> @redthing1, do you know of a way to do this in bash? i dont particularly like adding yet another script to my ~/.local/bin

You literally just run it once and paste the output into your config file. You do not need to add the script anywhere. It's a tool to convert the xresources to kitty format.

**Comment by Ahwxorg** [comment]
yeah but i want to get colors from pywal when i change my wallpaper :p
